### DISCUSSION TITLE
Dear Technical Support Team,

### DISCUSSION THREAD

**Discussion by KawhiWei** [nanoframework/Home#1255] [General]
### Library/API/IoT binding

nanoFramework.System.Device.Wifi-1.5.54

### Visual Studio version

VS2022(17.5.3)

### .NET nanoFramework extension version

2022.3.0.22

### Target name(s)

ESP32_PSRAM_REV0

### Device capabilities

Com3

### Description

I am an ESP32 user and I have encountered some issues while trying to connect to WiFi using NanoFramework. I have followed the documentation provided by NanoFramework for configuration and development, but I am still having trouble connecting to WiFi. Furthermore, I am unable to see any error messages or exception information, which has caused some difficulties in my development work.
Therefore, I would like to inquire if there are any solutions available to help me address this problem. If so, could you please provide some guidance or suggestions so that I can better complete my work?
Thank you very much for your help and support. I look forward to hearing back from you soon.

**Comment by alberk8** [comment]
What is the firmware load and what is the hardware revision version.  [ More info here](https://github.com/nanoframework/Home/discussions/1249)
